### ISSUE TITLE
Not possible to build it anymore with NB 8.2

### ISSUE THREAD

**Issue by Chris2011** [GeertjanWielenga/KendoNetBeans#3]
It is not possible to build the plugin anymore, because of the message, that this plugin is not a friend of the API which you are using.

**Comment by GeertjanWielenga** [comment]
There's a lot that needs to be done before it is properly useful, not just this.
Gj


Sent from Yahoo Mail for iPhone


On Wednesday, September 13, 2017, 12:37 PM, Chrizzly <notifications@github.com> wrote:


It is not possible to build the plugin anymore, because of the message, that this plugin is not a friend of the API which you are using.

**Comment by Chris2011** [comment]
Sure, but no one can contribute to the plugin anymore, you too, because we are not able to build the plugin anymore, because of the problem with the friend module of an existing core module.

**Comment by Chris2011** [comment]
Of course as a quick fix, we can add it to the core by our own, but this is only a workaround for testing the plugin again,

**Comment by GeertjanWielenga** [comment]
Yup, that's the status right now. Honestly, far down my list of priorities at the moment.
Gj


Sent from Yahoo Mail for iPhone


On Wednesday, September 13, 2017, 2:00 PM, Chrizzly <notifications@github.com> wrote:


Sure, but no one can contribute to the plugin anymore, you too, because we are not able to build the plugin anymore, because of the problem with the friend module of an existing core module.

**Comment by Chris2011** [comment]
I can understand it, thx for the clarifcation, I only wanted to make it clear here.

**Comment by GeertjanWielenga** [comment]
Really appreciate your understanding of this right now. :-)
Gj


Sent from Yahoo Mail for iPhone


On Wednesday, September 13, 2017, 2:02 PM, Chrizzly <notifications@github.com> wrote:


I can understand it, thx for the clarifcation, I only wanted to make it clear here.
